endpoints/version.go

```diff
@@ -0,0 +1,32 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/golang/glog"
+)
+
+type versionModel struct {
+	Revision *string `json:"revision"`
```

Comment:
Any reason this is a pointer? Seems like it's always non-nil... and `json.Marshal` should work the same for all non-nil cases.

Reply:
Nope, you are right! just updated the review